### PR TITLE
Adding in multi running models

### PR DESCRIPTION
**Pull request by matdenkas**
You can test running multiple models like this:

```docker exec semaphore-core python3 src/semaphoreRunner.py -d ./VirginiaKey/12hr_VirginiaKey_wl_Estrada.json ./VirginiaKey/24hr_VirginiaKey_wl_Estrada.json```

### CODE REVIEW

File: src/semaphoreRunner.py [hunk 1]
```diff
@@ -191,10 +188,21 @@ def handle_successful_prediction(model_name: str, execution_time: datetime, resu
             log(inserted_results)
             log(inserted_results.data if inserted_results is not None else '')
     except:
-        log(Semaphore_Exception('ERROR:: An error occurred while trying to interact with series storage from semaphoreRunner'))
+        log(Semaphore_Exception('ERROR:: An error occurred while trying to interact with series storage from semaphoreRunner'))    
 
-    log('Semaphore fin!')
-    exit(0)
+
+def clean_and_check_dspec(dspec_path: str) -> str:
```
Comment (lovelysandlonelys) [inline src/semaphoreRunner.py:194]: Nice to add a function for this!

**Review by lovelysandlonelys — APPROVED** [review]
Ran correctly and documentation looks good to me! One thing we might consider is also changing the file comment to mention that we can now run multiple models but I don't think it's necessary.

File: src/semaphoreRunner.py [hunk 2]
```diff
@@ -122,6 +119,8 @@ def handle_failed_prediction(
                             input_gatherer: InputGatherer | None, 
                             toss: bool
                             ):
+    """Handles a failed predictions. This function handles semaphore in an unstable error state. It emmits a notification and
```
Comment (lovelysandlonelys) [inline src/semaphoreRunner.py:122]: Handles a failed predictions -> Handles failed predictions.

Comment (matdenkas) [reply src/semaphoreRunner.py:122]: Nice catch, however, we run these in sequence so this function would still only handle one instance. Unless ive misunderstood what you mean.

**Pull request merged**